tt1-blocks/readme.txt

```diff
@@ -0,0 +1,59 @@
+=== TT1 Blocks ===
+Contributors: wordpressdotorg
+Requires at least: 5.6
+Tested up to: 5.6
```

Comment:
[style.css](https://github.com/WordPress/theme-experiments/blob/master/tt1-blocks/style.css#L7-L8) suggests `5.3` for requires and tested versions — should we bump that there also?

Reply:
Ah good call. Yes, I'll update that.